src/pages/communities/index.tsx

```diff
@@ -12,10 +15,18 @@ export const getStaticProps: GetStaticProps = async ({
     const clientOptions = previewData as ClientConfig;
 
     const data = await Prismic.getByTypes({ clientOptions, lang, types: 'pwa-view-communities' });
+    const communities = await fetcher('/communities?ambassadorAddress=&country=&limit=8&name=&page=0&offset=0&orderBy=bigger:DESC&status=valid&type=all&state=valid');
+    const claimsArray = await fetcher(`/claims-location`);
+    const communitiesCountries = await fetcher(`/communities/count?groupBy=country`);
 
     return {
         props: {
             data,
+            fallback: {
+                '/claims-location': claimsArray,
+                '/communities/count?groupBy=country': communitiesCountries,
+                '/communities?ambassadorAddress=&country=&limit=8&name=&offset=0&orderBy=bigger:DESC&page=0&status=valid&type=all': communities
```

Comment:
'page' and 'type' don't exist. Unused parameters can be ignored.
Could be something like this
```suggestion
                '/communities?limit=8&offset=0&orderBy=bigger:DESC&status=valid': communities
```
right @dev-jotape ?

This also applies for the query above, line 18

Reply:
And state as well (or status not sure right now) those are required for the frontend to work properly, forgot to remove those from the request, will do that. Thanks

Reply:
The changes have been applied. Thanks